packet/models_tinkerbell.go

```diff
@@ -14,19 +16,29 @@ func (i InterfaceTinkerbell) Name() string {
 }
 
 func (d DiscoveryTinkerbellV1) LeaseTime(mac net.HardwareAddr) time.Duration {
-	return d.Network.InterfaceByMac(mac).DHCP.LeaseTime
+	leaseTime := d.Network.InterfaceByMac(mac).DHCP.LeaseTime
+	fmt.Println(leaseTime)
```

Comment:
You forgot to remove this `fmt.Println` (and the others) logging lines.

Reply:
PTAL